src/sfizz/BeatClock.cpp

```diff
@@ -175,32 +181,35 @@ void BeatClock::fillBufferUpTo(unsigned delay)
     for (unsigned i = fillIdx; i < delay; ++i)
         beatsPerBarData[i] = sig.beatsPerBar;
 
-    if (!isPlaying_) {
-        if (fillIdx < delay) {
-            fill(absl::MakeSpan(&beatNumberData[fillIdx], delay - fillIdx), 0);
-            fill(absl::MakeSpan(&beatNumberPosition[fillIdx], delay - fillIdx), 0.0f);
-        }
-        currentCycleFill_ = fillIdx;
```

Comment:
I changed this so that `currentCycleFill_` is set to `delay`. It seemed more appropriate.